src/users_db.rs

```diff
@@ -307,8 +313,10 @@ impl UsersDb {
     /// ```
     pub fn create(&self, user: &User) -> rusqlite::Result<User> {
```

Comment:
Do you think that is worthy to create an extra method 'createAdmin' for the first case? ... Perhaps way too generic. Need to read more code!